content/docs/user-guide/testing/best-practices/_index.md

```diff
@@ -89,14 +89,20 @@ Negative assertions are different from negative test cases. A negative test case
 
 It can be okay to include negative assertions when there is also at least one positive assertion, but they should not be relied on alone. Focus on identifying the expected output.
 
+### Disabling and skipping tests
+
+Ideally, all automated test failures prompt immediately fixing the broken feature (or updating the test when underlying expectations have changed). However, sometimes a tradeoff must be made to break a feature while enabling other functionality. In a case where functionality is known to be broken, automated tests can be temporarily "skipped" or disabled.
+
+Avoid immediately disabling tests when you see a failure. First consider what action the failure blocks, and why all contributors ignoring failure would be less risky. Whenever a test provided with O3DE gets disabled, please [cut an issue](https://github.com/o3de/o3de/issues/new/choose) to track fixing the feature and re-enabling the test.
```

Comment:
"the failure" makes this less accurate, it would actually be "all future failures"
Why is it appropriate to swap the order of "all contributors"? The original seemed to flow better. Though perhaps "less risky" should come first as risk is the important topic.
Adding remaining recommendations.

Reply:
Changing it to "all future failures" makes the intent much clearer. The suggestion I made was under the assumption that "...ignoring failure..." was referring to the failure that we are considering to disable. 

Consequently, I originally understood this to mean that "_I, the developer, should consider what action the failure blocks, and why ignoring the failure and choosing not to disable it, will be less risky for all_ other _contributors. By other contributors, I mean those who might pull code in which tests have been disabled, which poses a greater risk to them._"
-- This may not be what this text was trying to say, so I think overall we can improve this sentence so it's less ambiguous. 

When you say "all contributors", do you mean "all other contributors" who may encounter these failures. Or do you mean it to refer to the developer who's reading this, suggesting that they consider simply ignoring all future failures. If the latter, I'd suggest removing "contributors" altogether, and opt to use "you". 
As a suggestion, you can say: 
> First consider what action the failure blocks, and why it's less risky if you simply ignore all future failure.

How does this sound?